craftassist/servermgr/app.py

```diff
@@ -42,29 +40,9 @@
     run_sh_gz_b64 = b64encode(gzip.compress(txt)).decode("utf-8")
     run_flat_sh_gz_b64 = b64encode(gzip.compress(txt_flat)).decode("utf-8")
 
-logging.info("Connecting to redis at {}".format(os.environ["REDIS_URL"]))
-rconn = redis.from_url(os.environ["REDIS_URL"])
-
 
 @app.route("/")
-@app.route("/acl2020demo")
-def homepage():
-    logging.info(
-        "cookie instance_id={instance_id}, timestamp={timestamp}".format(
-            instance_id=flask.request.cookies.get("instance_id", ""),
-            timestamp=flask.request.cookies.get("timestamp", ""),
-        )
-    )
-    return flask.render_template("index.html")
-
-
 @app.route("/launch", methods=["GET", "POST"])
-# The limiter ensures that the web server will handle at most 60 requests of
-# each unique ip address or 100 requests in total per mintue to prevent
-# malicious attack. e.g. one user may make too many 'launch instances' requests
-# to flood ECS with containers
-@ratelimit(max_requests=60, seconds=60, key_func="ip")
-@ratelimit(max_requests=100, seconds=60, key_func="global")
```

Comment:
Any reason we are removing the rate limiter ?

Reply:
Initially it was removed for development purpose. Then I realized if we are doing container pool on Mephisto side, we might need to change the rate limiter for a bit. so removed it for now.

Don't have strong opinion here, can add it back if y'all want.